packages/frontend/src/pages/campaigns/index.tsx

```diff
@@ -14,11 +14,11 @@ import { useSearch } from "../../hooks/useSearch";
 
 const ORDERING_OPTIONS = [
     {
-        label: "Latest",
+        label: "Newest",
         value: 1,
```

Comment:
I'd still highly prefer to see these as strings as it's much clearer in the code.

Reply:
what do you mean by seeing this as strings, aren't they strings?

Reply:
No I mean the values. Values could be `newest-first` and `latest-first` or something like that and then in the sorting function you can type the ordering to be `"newest-first" | "oldest-first"` so that it's overall a bit clearer.

This makes us avoid using numbers as ordering options and makes the solution a bit more expressive.

Also, we really should support i18n in the select options through the `t` function.

Reply:
so instead the value being 1 it should be something like "newest-first"?

Reply:
I'll add i18n support, tks.

Reply:
Typescript has support for enums. We can do something like this:

<img width="334" alt="image" src="https://user-images.githubusercontent.com/5664434/226908928-d63a2f5b-b823-4224-a4a3-ba4e6f569414.png">

Reply:
Yeah but I find that to be a more convoluted way of just having the strings in a way

Reply:
Let's use this for now. Updated here: bbdc10b8a89e819d5c2c99714389ec3bb4fa6352

Reply:
Added translations as well: 7b03fc9ade46d439806daf392105c4e4792e4e10